Makefile.toolchain.mk

```diff
@@ -108,6 +108,12 @@ build/ft-tmp-%/.package-stamp: \
 			mv $$tool $$tool-$(automake_version); \
 			ln -s $$tool-$(automake_version) $$tool; \
 		done
+ifeq ($(build_platform), linux)
+		find $(abspath $(@D)/package)/bin -type f -exec sed -i'' -e "s_/opt/python27-64/bin/python_$(PYTHON3)_g" {} +
+endif
+ifeq ($(build_platform), macos)
+		LC_CTYPE=C find $(abspath $(@D)/package)/bin -type f -exec sed -i '' -e "s_/opt/python27-64/bin/python_$(PYTHON3)_g" {} +
+endif
```

Comment:
This is testing right now, this regex should fix all of them to /usr/bin/env python3

Reply:
Awesome! Maybe we should just hardcode `python3` instead of using `$(PYTHON3)`? It should be reasonable to expect a `python3` symlink somewhere on the user's `PATH`.

Reply:
Right now that is basically what we are doing. $(PYTHON3) is pulled from config.mk, which is just python3. It doesn't have a path. Is there an advantage to hardcoding?

Reply:
It's meant to be user-overridable, so it could be an absolute path, which we don't want after `/usr/bin/env`.

Reply:
Fair enough, hard coded python3 it is